install/first_google_boot.sh

```diff
@@ -215,21 +219,42 @@ function extract_spinnaker_kube_credentials() {
     kube_zone=$MY_ZONE
   fi
 
+  if clear_metadata_to_file "kube_config" $config_path; then
```

Comment:
I was doing the same -- enable kube if you were able to extract out a path.
Since you mentioned it, if you specified kube but could not extract a path, that should manifest itself somewhere as a visible error. Either failure to startup (crash here) or from error logs in clouddriver saying kube is enabled but you dont have vaild credentials.